hover-provider-plugin/yarn.lock

```diff
@@ -0,0 +1,2777 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
using @next with this yarn.lock implies that we're fixing the package in all case

Reply:
Thanks, I'll substitute it with `@latest` version.